ENH: Add parsing of comma separated list for --castxml-start option

iMichka opened the pull request:
In gccxml one could input a comma-separated list for the starting declarations
to parse. The infrastructure is already there in castxml as there is a for loop on
opts.StartNames; but the inputed option was never parsed split.

bradking commented:
Thanks.  The current implementation supports repeating the option, though this is not covered in the test suite or documented.

Please revise the documentation to mention both ways of doing this and extend the test suite to cover them.

iMichka commented:
I rebased on master, and updated my code (I did not realise you could use --castxml-start multiple times at the first glance ...). I also added tests for the two cases by hijacking the castxml_test_gccxml_common function.

bradking commented:
Thanks.  I refactored the test infrastructure a bit and revised wording in the documentation.  See commit 73b947ec075e536535e9cc5bec84d1bed4d43d5b and its two immediate ancestors.

iMichka commented:
Thanks !
